.github/workflows/stage.yml

```diff
@@ -5,30 +5,42 @@ on:
   workflow_dispatch:
   push:
     branches:
-      - main
+      - master
     paths-ignore:
```

Comment:
@IlonaShishov you might consider trigger a workflow also if you're changing the workflow itself...

Reply:
you are correct, will make modifications, thanks